src/g3log/loglevels.hpp

```diff
@@ -145,44 +145,48 @@ namespace g3 {
    // Only safe if done at initialization in a single-thread context
    namespace only_change_at_initialization {
 
-      // This sets the logging level status enabled or disabled for a logging level.
-      // If the logging level is NEW it will also ADD the logging level to the set of logging levels.
-      void setLogLevel(LEVELS level, bool enabled_status);
+      /// add a custom level - enabled or disabled
+      void addLogLevel(LEVELS level, bool enabled);
 
-      /// for a custom level, 'MYL_LEVEL', previously not added the following is equivalent
-      /// setLogLevel(myLevel, true)
+      /// add a custom level - enabled
       void addLogLevel(LEVELS level);
 
+      /// reset all default logging levels to enabled
+      /// remove any added logging levels so that the only ones left are
+      ///  {DEBUG,INFO,WARNING,ERROR,FATAL}
+      void reset();
+   } // only_change_at_initialization
+
+
+   namespace log_levels {
       /// Enable log level >= log_level. 
       /// log levels below will be disabled
       /// log levels equal or higher will be enabled.
-      /// NOTE: If the level didn't exist prior then it will be added. This is similar to how
-      ///       'setLogLevel' works
-      void setHighestLogLevel(LEVELS level);
+      void setHighest(LEVELS level);
```

Comment:
Shouldn't the function rather be named `setLowest`? If I call `setLowest(WARNING)` I would expect `WARNING` to be the lowest active level - which it is then -, not the highest like with `setHighest(WARNING).` Or am I misunderstanding something?!